scripts/jobs/top_10_most_expensive_queries.sql

```diff
@@ -0,0 +1,34 @@
+-- Top 10 most expensive queries, last 7 days.
+SELECT
+  job_id,
+  user_email,
+  start_time,
+  end_time,
+  UNIX_MILLIS(end_time) - UNIX_MILLIS(start_time) runtime_millis,
+  query,
+  CASE
+    WHEN error_result IS NOT NULL THEN TRUE
+  ELSE
+  FALSE
+END
+  query_failed,
```

Comment:
This column may be better named as `is_error` given it's a `BOOL`

Reply:
I went with `has_error`.

Reply:
That should be fine.